test/integration/targets/elb_target/playbooks/roles/elb_lambda_target/tasks/main.yml

```
@@ -19,7 +17,6 @@
 
     - name: "create or update service-role for lambda"
       iam_role:
-        <<: *aws_connection_info
         name: ansible_lambda_execution
```

```suggestion
        name: "ansible-test-sts-{{ resource_prefix }}-lambda-execution"
```
To match the security policy in aws-terminator please.

---

@jillr

ansible-test-sts feels wrong as a prefix, this role has nothing to do with sts testing.

Running the tests using ansible-test will [automatically generate a prefix of "ansible-test-"
](https://github.com/ansible/ansible/blob/devel/test/lib/ansible_test/_internal/cloud/__init__.py#L388)

Rather than renaming to `ansible-test-sts...` how would you feel about renaming it to `{{ resource_prefix }}-elb-target-lambda-execution` and updating terminator to clean ansible-test-* ?

---

@jillr I've submitted #61806.  Once that's in I'll update this to `{{ resource_shortprefix }}-lambda-execution` as per the various discussions.  (`ansible-test-123456-lambda-execution`)

---

Renamed in latest version.  If #61806 goes through first I'll remove the temporary definition of resource_shortprefix from this PR.  If this one goes through first I'll clean it up in #61806